index.js

```diff
@@ -4037,27 +4043,30 @@
   S.parseInt =
   def('parseInt', {}, [$.Integer, $.String, $Maybe($.Integer)], parseInt_);
 
-  //# parseJson :: (a -> Boolean) -> String -> Maybe b
+  //# parseJson :: (Any -> Boolean) -> String -> Maybe a
```

Comment:
This may be beyond the scope of this PR.  But this change could really benefit new users who want to dive in and maybe aren't sure how to bridge the gap from unsafe javascript to sanctuary.  Perhaps functions that accept `Any` and safely return a known type (via Maybe/Either or a default value) could be bundled together in a section: "Migrating from Unsafe Javascript" or "Integrating with an existing Javascript codebase", or something to that affect.  It need not be at the top of the file, but it could maybe be linked to quite early on to reassure users that there is an on ramp.

I'm thinking all the `parse` functions, `get`, `gets`, and potentially others.